config/holes.toml

```diff
@@ -214,6 +214,33 @@ preamble = '''
     and execute each program in turn.
 '''
 
+['Catalan Constant']
+category = 'Mathematics'
+experiment = -1
+links = [
+   { name = 'OEIS A006752', url = 'https://oeis.org/A006752' },
+   { name = 'Wikipedia',    url = 'https://en.wikipedia.org/wiki/Catalan%27s_constant' },
+]
+preamble = '''
+<p>Print the Catalan constant to the first 1,000 decimal places.
+'''
+
+['Catalan Numbers']
+category = 'Sequence'
+experiment = -1
+links = [
+    { name = 'OEIS A000108', url = '//oeis.org/A000108' },
```

Comment:
https: ?

Reply:
Looking at the other entries, it doesn’t seem to be necessary

happy to change it in case

Reply:
JRaspass prefers to omit it

Reply:
Golf all the things! :golf: